File: backoffice/FondaDomain/Domain/NounBaseEntity.cs
using System;

namespace com.ds201625.fonda.Domain
{
	/// <summary>
	/// Entidad con nombre (Sustantivo)
	/// </summary>
	public class NounBaseEntity : BaseEntity
	{

		/// <summary>
		/// El nombre de la entidad
		/// </summary>
		private string _name;

		/// <summary>
		/// Construntor
		/// </summary>
		public NounBaseEntity () : base () { }

		/// <summary>
		/// Obtiene y asigna el nombre de la entidad
		/// </summary>
		/// <value>El nombre</value>
		public virtual string Name
		{
			get { return _name; }
			set { _name = value; }
		}

	}
}

File: frontoffice/app/src/main/java/com/ds201625/fonda/domains/NounBaseEntity.java
package com.ds201625.fonda.domains;

import com.ds201625.fonda.domains.BaseEntity;

/**
 * Clase base Nombrada.
 */
public abstract class NounBaseEntity extends BaseEntity {

    /**
     * Sustantivo de la entidad
     */
    private String name;

    public NounBaseEntity() {
        super();
    }

    /**
     * Obtiene el sustantivo de la entidad
     * @return Sustantivo
     */
    public String getName() {
        return name;
    }

    /**
     * Asigna el susutantivo a la entidad
     * @param name el sustantivo
     */
    public void setName(String name) {
        this.name = name;
    }
}
